capstone-app/src/app/modules/history/wiki-media/wiki-media.component.spec.ts

```diff
@@ -53,12 +125,39 @@ describe('WikiMediaComponent', () => {
 
     it('displays location history', () => {
       component.getResults('Stillwater, Oklahoma');
-      expect(component.history).toEqual(testWikiResult.parse.text["*"]);
+      expect(component.history).toEqual(testWikiServiceResponse.history);
     });
 
     it('displays correct location title', () => {
```

Comment:
For this test and the "displays location history one" it makes more sense to either 1) switch the name of the test or 2) look at the DOM to see if the info actually is displayed